sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -59,7 +59,8 @@
                     "key": "html3",
                     "type": "htmlelement",
                     "input": false,
-                    "tableView": false
+                    "tableView": false,
+                    "hideLabel": true
```

Comment:
The hide label being in the change seems to be weird. As the hide label is already enabled. 

![image](https://github.com/bcgov/SIMS/assets/54600590/8eff832e-a0e2-4bfb-859c-513bcf8a4ff9)

Reply:
I believe these new tags are part of the formio generation. I too tried to mimic and it seems to generate it.

Reply:
@katrina-aot-git raised the concern to me also during development, it seems that it seems the editor causing it as an out-of-box behavior.
I was able to reproduce using the code from the main branch, just changing the order of the hidden fields and saving it.